cadence/transactions/bridge/admin/metadata/set_bridged_nft_collection_display_view.cdc

```diff
@@ -0,0 +1,80 @@
+import "MetadataViews"
+
+import "FlowEVMBridgeResolver"
+
+/// Builds a thumbnail file based on the provided thumbnail file type identifier and optional IPFS file path
+///
+access(all)
+fun buildFile(_ uri: String, _ fileTypeIdentifier: String, _ ipfsFilePath: String?): {MetadataViews.File} {
```

Comment:
Maybe these buildFile and buildSocials utility functions should be utility functions on the smart contract instead of copying them into each transaction?

Reply:
Added in [2acad5b](https://github.com/onflow/flow-evm-bridge/pull/116/commits/2acad5b8e7a25da8132e29887970d07b246f9be9)